.github/workflows/linux_conditional_compilation.yml

```diff
@@ -76,44 +82,267 @@ jobs:
+      MODELS_PATH: /__w/openvino/openvino/testdata
+      CCACHE_DIR: /mount/caches/ccache/ubuntu22_x86_64_cc_Release
+      CCACHE_TEMPDIR: /__w/openvino/openvino/ccache_temp
+      CCACHE_MAXSIZE: 20G
+
+    steps:
+      - name: Install git
+        run: apt-get update && apt-get install --assume-yes --no-install-recommends git ca-certificates git-lfs
+
+      - name: Clone OpenVINO
+        uses: actions/checkout@v4
+        with:
+          path: ${{ env.OPENVINO_REPO }}
+          submodules: 'true'
+
+      - name: Clone test models
+        uses: actions/checkout@v4
+        with:
+          repository: 'openvinotoolkit/testdata'
+          path: ${{ env.MODELS_PATH }}
+          lfs: 'true'
+
+      - name: Download selective build statistics package
+        uses: actions/download-artifact@v3
+        with:
+          name: openvino_selective_build_stat
+          path: ${{ env.INSTALL_TEST_DIR }}
 
-      - name: Build with CC ON
-        run: cmake --build ${BUILD_DIR} --parallel --config ${{ env.CMAKE_BUILD_TYPE }} --target openvino_intel_cpu_plugin openvino_ir_frontend
+      #
+      # Dependencies
+      #
+
+      - name: Install build dependencies
+        run: bash ${OPENVINO_REPO}/install_build_dependencies.sh
+
+      #
+      # Build
+      #
+
+      - name: CMake configure - CC ON
+        run: |
+          cmake \
+            -DBUILD_SHARED_LIBS=OFF \
+            -DENABLE_CPPLINT=OFF \
+            -DSELECTIVE_BUILD=ON \
+            -DENABLE_LTO=OFF \
+            -DSELECTIVE_BUILD_STAT=${SELECTIVE_BUILD_STAT_DIR}/*.csv \
+            -DCMAKE_C_COMPILER_LAUNCHER=${{ env.CMAKE_C_COMPILER_LAUNCHER }} \
+            -DCMAKE_CXX_COMPILER_LAUNCHER=${{ env.CMAKE_CXX_COMPILER_LAUNCHER }} \
+            -S ${OPENVINO_REPO} \
+            -B ${BUILD_DIR}
+
+      - name: Cmake build -  CC ON
+        run: cmake --build ${BUILD_DIR} --parallel 8 --target benchmark_app
+
+      - name: Show ccache stats
+        run: ccache --show-stats
 
-      - name: Use OpenVINO after CC
+      - name: Run with CC-ed runtime
         run: |
           ${OPENVINO_REPO}/bin/intel64/Release/benchmark_app -niter 1 -nireq 1 \
             -m ${MODELS_PATH}/models/test_model/test_model_fp32.xml -d CPU
+
+  CPU_Functional_Tests:
+    name: CPU functional tests
+    needs: Build
+    defaults:
+      run:
+        shell: bash
+    runs-on: aks-linux-8-cores
+    container:
+      image: openvinogithubactions.azurecr.io/dockerhub/ubuntu:22.04
+    env:
+      DEBIAN_FRONTEND: noninteractive # to prevent apt-get from waiting user input
+      INSTALL_TEST_DIR: /__w/openvino/openvino/install/tests
+      PARALLEL_TEST_SCRIPT: /__w/openvino/openvino/install/tests/src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py
+      PARALLEL_TEST_CACHE: /__w/openvino/openvino/install/tests/test_cache.lst
+
+    steps:
+      - name: Create Directories
+        run: mkdir -p ${INSTALL_TEST_DIR}
```

Comment:
No need as it will be created by the download action.

Reply:
removed